Marlin/src/lcd/extui/lib/mks_ui/wifi_module.cpp

```diff
@@ -109,8 +115,9 @@ extern IP_PARA ipPara;
 extern CLOUD_PARA cloud_para;
 
 extern bool once_flag, flash_preview_begin, default_preview_flg, gcode_preview_over;
+extern char flash_dma_mode;
 
-extern uint8_t bmp_public_buf[17 * 1024];
+extern uint8_t bmp_public_buf[14 * 1024];
```

Comment:
I have seen this same extern over and over.

It would be better to declare it one time in a header file, then include that header everywhere.
The same applies to other `extern` variables which you use from multiple files.